pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
+type Scale int
```

Comment:
I think it needs to be public so that arbitrary values can be passed to the Scale methods (or was your goal to prevent that?)